RELEASE_NOTES.rst

```diff
@@ -1,3 +1,13 @@
+coala-bears 0.8.3
+=================
+
+This bugfix release fixes the following issues:
+
+- YapfBear handles files with syntax errors gracefully.
+  (https://github.com/coala-analyzer/coala-bears/issues/750)
+- ESLintBear doesn't fail when eslint fails anymore.
```

Comment:
hm now that I think about it this seems to be even worse than before - errors should not pass silently